validation/roomValidation.js

```diff
@@ -0,0 +1,17 @@
+const Joi = require('joi');
```

Comment:
Please, also here be mindful of spacing and how you add it. Also, please follow the structure we discussed and add the validation inside a folder together with the route.